docs/user-guide.md

```diff
@@ -752,6 +753,14 @@ The Spark Operator provides limited support for resource quota enforcement using
 
 If you are running Spark applications in namespaces that are subject to resource quota constraints, consider enabling this feature to avoid driver resource starvation. Quota enforcement can be enabled with the command line arguments `-enable-resource-quota-enforcement=true`. It is recommended to also set `-webhook-fail-on-error=true`.
 
+## Running multiple instances of the operator within the same k8s cluster
+
+If you need to run multiple instances of the operator within the same k8s cluster. Therefore, you need to make sure that the running instances should not compete for the same custom resources or pods. You can achieve this:
+
+* By specifying custom labels on resources by defining for each instance of the operator a different `-label-selector-filter (e.g. env=dev,app-type=spark)` option for custom resources such as `SparkApplication`.
```

Comment:
An alternative is to use the flag `namespace` to limit the operator to watching resources in a single namespace.

Reply:
As discussed above. It is ok for you ?